modules/ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: PROCEDURE
+[id="ztp-configuring-the-hub-cluster-to-use-unauthenticated-registries_{context}"]
+= Configuring the hub cluster to use unauthenticated registries
+
+You can configure the hub cluster to use unauthenticated registries.
+
+.Procedure
+
+. Edit the `AgentServiceConfig` CR to include the `unauthenticatedRegistries` parameter:
+
+[source,terminal]
+----
+$ apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+  name: agent
+spec:
+  unauthenticatedRegistries:
+  - example.registry.com
+  - example.registry2.com
+  ...
+----
```

Comment:
It might be a good idea to match the format of this procedure https://55352--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.html#enabling-assisted-installer-service-on-bare-metal_ztp-preparing-the-hub-cluster

```suggestion
. Update the AgentServiceConfig CR by running the following command:

[source,terminal]
----
$ oc edit AgentServiceConfig
---

. Add the `unauthenticatedRegistries` field in the CR:

[source,terminal]
----
apiVersion: agent-install.openshift.io/v1beta1
kind: AgentServiceConfig
metadata:
  name: agent
spec:
  unauthenticatedRegistries:
  - example.registry.com
  - example.registry2.com
  ...
----
```

Reply:
+1 please make this change

Reply:
Updated as suggested.